components/blocks/FilesToCards.vue

```diff
@@ -59,26 +61,229 @@
         </template>
       </DCard>
     </div>
+    <div v-else>
+      <div class="sort-bar">
+        <multiselect
+          class="item-a"
+          v-model="searchGroup"
+          :options="cardTags()"
+          :close-on-select="true"
+          :clear-on-select="false"
+          :preserve-search="true"
+          :multiple="true"
+          placeholder="Select tags to filter by"
+          :allow-empty="true"
+        >
+        </multiselect>
+        <button
+          class="item-b button is-normal is-warning"
```

Comment:
I don't see this button in the app

Reply:
The QA link appears to be only semi caught up to date with my commits?